rustls/src/suites.rs

```diff
@@ -176,10 +190,7 @@ pub(crate) fn choose_ciphersuite_preferring_server(
 
 /// Remove the suites in `all` that are
 /// incompatible with `SignatureAlgorithm` `sigalg`.
-pub(crate) fn reduce_given_sigalg(
-    all: &mut Vec<SupportedCipherSuite>,
-    sigalg: SignatureAlgorithm,
-) {
+pub(crate) fn reduce_given_sigalg(all: &mut Vec<SupportedCipherSuite>, sigalg: SignatureAlgorithm) {
```

Comment:
Nit: this belongs in the earlier commit.

Reply:
The function is gone now!